src/foam/u2/view/MultiChoiceView.js

```diff
@@ -261,7 +260,7 @@ foam.CLASS({
                   .addClass(self.myClass('innerFlexer'))
                   // NOTE: This should not be the way we implement columns.
                   .style({
-                    'width': `${100 / self.NUM_COLUMNS}%`
+                    'width': self.isVertical ? '100%' : `${100 / self.NUM_COLUMNS}%`
```

Comment:
This should just be handled by flex and cardSelectView but for now it's fine

Reply:
not sure how this should be handled by cardselectview here, can u please explain?

Reply:
I'm just not sure what the innerFlexer is doing, Couldn't this style tag just be added to the div that adds cardSelectView?

Reply:
it doesnt look like the innerflexer is doing anything except add a padding, which could be moved to cardselectview
but i think the style tag needs to be there instead of on the cardselectview cuz the E() adds a parent div to the cardselectview

Reply:
Yeah, We can leave it as is for now. It just adds an extra div I think because we shouldn't need the parent div above the card select view

Reply:
ok, i just moved the innerflexer padding to the card, leaving the style tag for now